### Kuon kuon_control/ProductInfo message

string  version_string    # hardware version string
int64   maj               # hardware major version
int64   min               # hardware minor version
int64   rev               # hardware revision

string  product_name      # product name
uint32  product_id        # product unique id
string  desc              # product description
